assets/js/components/notifications/CoreSiteBannerNotification.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * CoreSiteBannerNotification component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import BannerNotification from './BannerNotification';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+
+const { useDispatch } = Data;
+
+const CoreSiteBannerNotification = ( { notification } ) => {
```

Comment:
Please, add propTypes for this component.

Reply:
https://github.com/google/site-kit-wp/pull/4349/commits/2a19ad690da30f1cd11799d7c74d1332bf6b1798